config.h.in

```diff
@@ -15,5 +36,35 @@
 /* Define to 1 if `st_rdev' is a member of `struct stat'. */
 #undef HAVE_STRUCT_STAT_ST_RDEV
 
-/* Define to 1 if `st_birthtimespec' is a member of `struct stat'. */
-#undef HAVE_STRUCT_STAT_ST_BIRTHTIMESPEC
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#undef HAVE_SYS_STAT_H
+
+/* Define to 1 if you have the <sys/sysmacros.h> header file. */
+#undef HAVE_SYS_SYSMACROS_H
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#undef HAVE_SYS_TYPES_H
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#undef HAVE_UNISTD_H
+
+/* Define to the address where bug reports for this package should be sent. */
+#undef PACKAGE_BUGREPORT
+
+/* Define to the full name of this package. */
+#undef PACKAGE_NAME
+
+/* Define to the full name and version of this package. */
+#undef PACKAGE_STRING
+
+/* Define to the one symbol short name of this package. */
+#undef PACKAGE_TARNAME
+
+/* Define to the home page for this package. */
+#undef PACKAGE_URL
+
+/* Define to the version of this package. */
+#undef PACKAGE_VERSION
+
+/* Define to 1 if you have the ANSI C header files. */
+#undef STDC_HEADERS
```

Comment:
Are there result of `autoheader`?
I'm not sure the necessity.
I would like to limit it only to what I need.
Do you know what is the advantage of write everything?

Reply:
Sorry, I'm not sure about that.
I just run autoreconf to re-generate configure, because I've edited configure.ac in the first commit of this PR. I did not expect to re-generate config.h.in.

@kou Do you know about this? Could you help us?

Reply:
If @ksss wants to reduce these diffs, it's better that @ksss accepts only changes in `configure.ac` and `src/file-stat.c` and run `autoreconf` on @ksss's environment after it's merged.
These generated files (`configure` and `config.h.in`) have different contents when we use different version of GNU Autotools. It seems that @ksss and @okkez use different version of GNU Autotools.